Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -189,6 +199,28 @@ class BrowserViewController: UIViewController {
             }
             footer.addSubview(footerBackground!)
         }
+        
+        if showTopTabs {
+            if topTabsContainer.subviews.count == 0 {
```

Comment:
Nit: I'm a bit wary on inferring that the view controller has been added by checking the subview count. This seems prone to error since added a subview has the side effect of this not working correctly. A more direct check could be to check `topTabsViewController.parentViewController == nil`.